src/main/java/racingcar/Racingcar.java

```diff
@@ -0,0 +1,26 @@
+package racingcar;
+
+import racingcar.ui.InputView;
+
+import java.util.*;
+
+public class Racingcar {
+    private static final int MAX_INPUT_VALUE = 10000;
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        int numberOfCar = InputView.getInstance().inputNumberOfCar(scanner);
+        int numberOfAttempt = InputView.getInstance().inputNumberOfAttempt(scanner);
```

Comment:
InputView는 사용자 입력을 처리하는 역할로 보이네요
스캐너를 InputView 내부에서 관리하면 어떨까요?
```suggestion
        int numberOfCar = InputView.getInstance().inputNumberOfCar();
        int numberOfAttempt = InputView.getInstance().inputNumberOfAttempt();
```